src/main.py

```diff
@@ -407,6 +410,10 @@ async def main() -> None:
                             logging.info(f"data after json dump {data_json}")
                             event = CloudEvent(attributes, data_json)
                             headers, body = to_structured(event)
+                            info = {
+                                key: data[key]
+                                for key in ["instrument", "groupId", "detector"]
+                            }
```

Comment:
The formatting here looks a bit odd; it took me a moment to parse that it was a dict comprehension and not a dict of two items. Can this fit on one line?

Reply:
Yes it can. That was `black`'s choice, though.